### DISCUSSION TITLE
Minor UI bug on the Pull Request screen, iOS application

### DISCUSSION THREAD

**Discussion by Scorpikk** [community/community#76630] [Mobile]
### Select Topic Area

Bug

### Body

Noticed a minor bug in Update branch section.

Version: 1.139.0 (115734377)
Device: iPad (iOS 17.1.1), landscape orientation

![bug](https://github.com/community/community/assets/7117206/f00bd338-7f53-42a9-96a5-239041a85a4b)

**Comment by taki-on** [comment]
Thank you for the report @Scorpikk! Sorry for the inconvenience and a fix is soon out to the production :) Please stay tuned :)
